src/ray/gcs/gcs_server/stats_handler_impl.cc

```diff
@@ -34,7 +35,7 @@ void DefaultStatsHandler::HandleAddProfileData(const AddProfileDataRequest &requ
     GCS_RPC_SEND_REPLY(send_reply_callback, reply, status);
   };
 
-  Status status = gcs_table_storage_->ProfileTable().Put(UniqueID::FromRandom(),
+  Status status = gcs_table_storage_->ProfileTable().Put(ids_[cursor_++ % ids_.size()],
```

Comment:
I just need some clarification answer here. When you store data in this way, and run `ray timeline`, does it work as expected?

Reply:
Also, please comment why we are doing this.

Reply:
> I just need some clarification answer here. When you store data in this way, and run `ray timeline`, does it work as expected?

![image](https://user-images.githubusercontent.com/13081808/93695137-fcc05600-fb45-11ea-82d9-f846177c16a6.png)
The first time I used `ray timeline`, I'm not sure if it work as expected.